src/chopper_layout.cpp

```diff
@@ -16,79 +16,132 @@ void set_up_subparser_layout(seqan3::argument_parser & parser, chopper::layout::
-                      "when -determine-num-bins is given.");
-
-    parser.add_option(config.num_hash_functions, 's', "num-hash-functions",
-                      "The number of hash functions for the IBFs.");
-
-    parser.add_option(config.fp_rate, 'p', "false-positive-rate",
-                      "The desired false positive rate of the IBFs.");
+    parser.add_option(config.num_hash_functions,
+                      '\0', "num-hash-functions",
+                      "The number of hash functions to use when building the HIBF from the resulting layout. "
+                      "This parameter is needed to correctly estimate the index size when computing the layout.");
 
-    parser.add_option(config.alpha, 'a', "alpha",
-                      "The scaling factor to influence the number of merged bins.", seqan3::option_spec::advanced);
+    parser.add_option(config.fp_rate,
+                      '\0', "false-positive-rate",
+                      "The false positive rate you aim for when building the HIBF from the resulting layout. "
+                      "This parameter is needed to correctly estimate the index size when computing the layout.");
 
-    parser.add_option(config.output_filename, 'o', "outfile",
-                      "An output file name for the binning results.");
+    parser.add_option(config.output_filename, '\0', "output-file", "A file name for the resulting layout.");
 
     using aggregate_by_type = std::remove_cvref_t<decltype(config.aggregate_by_column)>;
-    parser.add_option(config.aggregate_by_column, 'y', "aggregate-by",
+    parser.add_option(config.aggregate_by_column,
+                      '\0', "aggregate-by",
                       "Which column do you want to aggregate your files by? Start counting your columns from 0!",
                       seqan3::option_spec::hidden,
                       seqan3::arithmetic_range_validator{aggregate_by_type{2},
                                                          std::numeric_limits<aggregate_by_type>::max()});
 
-    parser.add_section("HyperLogLog Sketches");
-    parser.add_line("To improve the _layouting, you can estimate your sequence similarities using HyperLogLog sketches.");
-    parser.add_line("\n");
-
-    parser.add_option(config.max_ratio, 'm', "max-ratio",
-                      "[HLL] The maximal cardinality ratio in the clustering intervals.",
-                      seqan3::option_spec::standard,
-                      seqan3::arithmetic_range_validator{0.0, 1.0});
-
-    parser.add_option(config.num_threads, 't', "num-threads",
-                      "[HLL] The number of threads to use to compute merged HLL sketches.",
+    parser.add_option(config.num_threads,
+                      '\0', "threads",
+                      "The number of threads to use. Currently, only merging of sketches is parallelized "
+                      "(see option --rearrange-user-bins).",
```

Comment:
It's not clear to me whether this `merging of sketches` is always done, or I need to also pass `--rearrange-user-bins`.

I.e., does `--threads` have an effect if I do not pass `--rearrange-user-bins`?

Reply:
yes it does not have an effect then. I changed the description